scanners/android/icon.go

```diff
@@ -105,7 +99,7 @@ func lookupIcons(projectDir string, basepath string) ([]string, error) {
 	for _, manifestPath := range manifestPaths {
 		icons, err := lookupIconName(manifestPath)
 		if err != nil {
-			return nil, err
+			analytics.LogInfo("android-icon-lookup", analytics.DetectorErrorData("android", err), "Failed to lookup android icon from manifest")
```

Comment:
missing continue

Reply:
append seems to be working without an explicit continue, but maybe the intent is clearer that way?

Reply:
i think so